docs/SCRIPTS.md

```diff
@@ -0,0 +1,39 @@
+# Tech Lead Scripts
+
+The following scripts are used for local development and managing deployments to AWS. Valid AWS admin credentials are required.
+
+`reset-dependencies.sh`
```

Comment:
Could we make the delineation of the script name/description pairs easier to see? Perhaps line markers in-between the pairs or some other kind of formatting would help.

Reply:
This was also suggested by @adunkman , the latest version is in a table: https://github.com/flexion/ef-cms/blob/develop/docs/SCRIPTS.md